src/routes/blog/developer-journey/timeleap-protocol.tiramisu

```diff
@@ -0,0 +1,135 @@
+title { Timeleap: The Communication Protocol }
+
+We've talked about Timeleap giving you the tools to create a distributed
+computation network link { to =
+/blog/developer-journey/introduction-to-timeleap, here}. In this article, we
+are connecting the dots: Timeleap, Sia, and the rules of the network.
+
+Think of Timeleap protocol as the complete rule book and blueprint to implement
+what we call a inlineCode { Timeleap App }. Timeleap has created its
+communication protocol to ensure not only that every message is reliable,
+secure, and fast, but also to ensure reproducibility and compatibility in
+different implementations. We have a reference implementation of this blueprint,
+which is our broker-worker model. Others can study the blueprint and implement
+their own, which you can check out link { to =
+https://github.com/TimeleapLabs/timeleap, here }.
```

Comment:
Still not there. If we talk about the Timeleap App, we should link to an example too. Not everyone needs to implement the worker/broker model. Yes, it's the reference implementation, but still, we should tell people they can bypass that and implement a standalone one with a link to a sample timeleap app. Or we don't tell them at all. Rarely does anyone need to implement the entire thing. Let me check with @Dirdmaster.